scraper.rb

```diff
@@ -21,18 +21,17 @@ def scrape_result_row(result_row)
 
   council_reference = fields[0].search('a')[0].inner_text
 
-  if (ScraperWiki.select("* from swdata where `council_reference`='#{council_reference}'").empty? rescue true)
-    puts "Saving record " + council_reference
-
+  if (ScraperWiki.select("* from data where `council_reference`='#{council_reference}'").empty? rescue true)
```

Comment:
The swdata table is the default in the scraperwiki gem. In the oaf fork of that gem we change this to `data` for use with morph.io. So this should actually be working on morph, but not locally unless you have the oaf fork installed as you default scraperwiki gem. In other words, your change is good, and I'll add a gemfile ;-)